readme.md

```diff
@@ -106,6 +106,7 @@ Colors can be changed by using [`zstyle`](http://zsh.sourceforge.net/Doc/Release
 - `git:arrow` (cyan) - For `PURE_GIT_UP_ARROW` and `PURE_GIT_DOWN_ARROW`.
 - `git:branch` (242) - The name of the current branch when in a Git repository.
 - `git:branch:cached` (red) - The name of the current branch when the data isn't fresh.
+- `git:branch:dirty` (208) - The asterisk showing the branch is dirty.
```

Comment:
I think it should be `git:dirty`. While the symbol is shown with the branch, it's not really related to the branch name.

Reply:
Can you find a brown color that is a bit less saturated/dimmed?

Reply:
Thanks for your feedback! 

I changed my commit, I hope the color fits this time. Otherwise you may suggest one and I can implement it.